kymatio/utils.py

```diff
@@ -0,0 +1,57 @@
+from sklearn.base import BaseEstimator, TransformerMixin
+
+import torch
+
+
+class ScatteringTransformer(BaseEstimator, TransformerMixin):
+
+    def __init__(self, S, signal_shape, frontend='numpy'):
+        """Creates an object that is compatible with the scikit-learn API
+        and implements the .transform method.
+        
+        Parameters
+        ==========
+
+        S: an instance of Scattering1D, Scattering2D or Scattering3D
+            This instance is called by the transformer
+
+        signal_shape: tuple of ints
+            The shape of one sample. Scikit-learn convention is to work with
+            2D matrices only, of shape (n_samples, n_features). Data is
+            delivered in this way and has to be reshaped before transform
+
+        Output
+        ======
+        Y: ndarray, shape (n_samples, n_scattering_features)
+            The scattering coefficients, raveled in C order
+        """
+
+        self.scattering = S
+        self.signal_shape = signal_shape
+        self.frontend = frontend
+
+        assert frontend in ['numpy','torch','tensorflow']
```

Comment:
I believe you need to use the tensor-flow reshape op, rather than `.reshape`

Reply:
we decided to remove tf compatability for this transformer